### ISSUE TITLE
orderbook minsize 0.00000000

### ISSUE THREAD

**Issue by raedah** [JoinMarket-Org/joinmarket#382]
I am seeing offers listed with a minsize of 0.00000000. The orderbook watcher should flag this as an invalid offer condition, since taking a transaction of size 0 is not a valid condition.

**Comment by chris-belcher** [comment]
I suppose the DUST_THRESHOLD constant could be used here https://github.com/JoinMarket-Org/joinmarket/blob/master/lib/taker.py#L333

**Comment by adlai** [comment]
"This issue will close once commit e7cb86d is merged into the 'master' branch."

Way ahead of you, shithub!
